### PR TITLE
Reformat config comments

### PR DESCRIPTION
**Pull request by MSpiller7**
#460 

Changed the comments to block comments, as per issue template. Also fixed one typo, and for any config files which are split up into sections (such as `config_objects`), the section headers were changed from single line comments to block comments for the sake of further delineating the sections.

### CODE REVIEW

**Review by Reonu — APPROVED** [review]
hot take: unless there's a problem with this (which I don't see) merge this now before it gets 8389363 conflicts

**Review by thecozies — APPROVED** [review]
I agree

**Review by Reonu — CHANGES_REQUESTED** [review]
@MSpiller7 As soon as you fix the existing conflicts we can merge

**Comment by MSpiller7** [comment]
hopefully i unborked it

**Review by MSpiller7 — COMMENTED** [review]
the only thing im a bit sus on is if that stuff in `config_world` is actually supposed to be gone, otherwise i think this fixes the conflict

File: include/config/config_audio.h [hunk 1]
```diff
@@ -22,6 +24,8 @@
...
 #define MAX_SIMULTANEOUS_NOTES_CONSOLE 24
 
-// Uses a much better implementation of reverb over vanilla's fake echo reverb. Great for caves or eerie levels, as well as just a better audio experience in general.
-// Reverb parameters can be configured in audio/synthesis.c to meet desired aesthetic/performance needs. Currently US/JP only. Hurts emulator and console performance.
-// #define BETTER_REVERB
+/** 
+ * Uses a much better implementation of reverb over vanilla's fake echo reverb. Great for caves or eerie levels, as well as just a better audio experience in general.
+ * Reverb parameters can be configured in audio/synthesis.c to meet desired aesthetic/performance needs. Currently US/JP only. Hurts emulator and console performance.
+ */
+// #define BETTER_REVERB
```
Comment (thecozies) [inline include/config/config_audio.h:31]: needs a newline at the bottom 
```suggestion
// #define BETTER_REVERB

```

**Comment by arthurtilly** [comment]
> the only thing im a bit sus on is if that stuff in `config_world` is actually supposed to be gone, otherwise i think this fixes the conflict

yeah it is supposed to be removed, i obseleted all that stuff with the automatic surface pool size pr

**Review by gheskett — APPROVED** [review]
Diff looks good to me

**Comment by thecozies** [comment]
Thanks @MSpiller7 !

**Comment by MSpiller7** [comment]
pog!!!

**Pull request merged**